node-unix-interface.md

```diff
@@ -0,0 +1,11 @@
+# The Node-Unix interface
+
+Imagine being able to build scripts that can fit into any pipeline of Unix
```

Comment:
While I know what you're talking about, anyone new to Unix command programming won't know what a "pipeline" is or what this sentence means. Can you generalize it to a wider audience and introduce pipelines later in the abstract?